uber/tests/conftest.py

```diff
@@ -52,6 +52,9 @@ def sensible_defaults():
     c.DEALER_BADGE_PRICE = 20
     c.PRICE_BUMPS = {}
 
+    # tests should turn this on to test the effects
+    c.HARDCORE_OPTIMIZATIONS_ENABLED = False
```

Comment:
Oh never mind, I saw your note in the description.  I must have not gotten to the PR where this was implemented yet.